test/e2e_node/remote/node_e2e.go

```diff
@@ -127,6 +127,7 @@ func osSpecificActions(args, host, workspace string) (string, error) {
 		args = prependGCPCredentialProviderFlag(args, workspace)
 		return prependCOSMounterFlag(args, host, workspace)
 	case strings.Contains(output, "ubuntu"):
+		args = prependGCPCredentialProviderFlag(args, workspace)
```

Comment:
The remote runner here is only relevant for GCE-based node e2e tests right? Wondering if it's at all possible to use this remote runner on ubuntu not on GCE. Otherwise LGTM

Reply:
@andrewsykim the node is ubuntu bootstrapped via GCE only
https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-credential-provider/1506976544550031360/build-log.txt
![image](https://user-images.githubusercontent.com/7712693/159939004-29e4a69f-a0b0-4305-be28-7b670a8d3f61.png)

Reply:
if this is universally true for our CI, this is probably ok as a starting point. the prow job does set `--provider=gce` so maybe plumbing that here and using that as the signal would be better in a follow up if this causes issues for non-gcp ubuntu jobs (if they even exist)